src/avkit.cs

```diff
@@ -345,7 +369,7 @@ interface AVPlayerViewControllerDelegate
 		[Export ("playerViewController:restoreUserInterfaceForPictureInPictureStopWithCompletionHandler:")]
 		void RestoreUserInterfaceForPictureInPicture (AVPlayerViewController playerViewController, Action<bool> completionHandler);
 
-		[NoiOS][NoMac]
+		[iOS (16,0)][NoMac, NoMacCatalyst, NoWatch]
```

Comment:
Same lets be consistent.
```suggestion
		[iOS (16,0)][NoMac][NoMacCatalyst][NoWatch]
```